rtengine/procparams.cc

```diff
@@ -129,22 +131,22 @@ RetinexParams::RetinexParams ()
     setDefaults ();
 }
 
-void RetinexParams::getDefaulttransmissionCurve(std::vector<double> &curve)
+void RetinexParams::getDefaulttransmissionCurve (std::vector<double> &curve)
```

Comment:
@Floessie please see this example I ran in `dev`:
```bash
astyle -n --options=rawtherapee.astylerc rtengine/procparams.cc
```
results in this one of many changes:
```diff
-void RetinexParams::getDefaulttransmissionCurve(std::vector<double> &curve)
+void RetinexParams::getDefaulttransmissionCurve (std::vector<double> &curve)
```

Of course I might be wrong about that being a function call since I don't speak C+, but it resembles one to me :)

Artistic Style Version 3.0.1